.circleci/config.yml

```diff
@@ -105,14 +93,8 @@ workflows:
   version: 2
   main:
     jobs:
-      - lint:
-          filters: *main_filters
-      - test:
-          filters: *main_filters
-          requires:
-            - lint
-      - build:
-          filters: *main_filters
-          requires:
-            - lint
-            - test
+      - full:
+          filters:
+            branches:
+              ignore:
+                - /\//
```

Comment:
Would you mind re-adding the comment about slashes? It's unclear why these are being ignored now.